spring-cloud-starter-stream-source-triggertask/README.adoc

```diff
@@ -29,7 +29,10 @@ A trigger task is a source that  dispatches `TaskLaunchRequest` messages that wi
 tasklauncher-* sink. The tasklauncher-* sink will then deploy a task and launch it.  An example of this using Spring
 Cloud Data Flow would look like this:
 
-`stream create foo --definition "triggertask --uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.0.BUILD-SNAPSHOT --fixed-delay=5 | tasklauncher-local"`
+``` 
+app import --uri http://bit.ly/1-0-1-GA-task-applications-maven
```

Comment:
Did some research into this and the actual solution is to configure the maven properties for the task-launcher-local.  
so the solution would be to remove the app import and add the following parameter to the task-launcher local:  `--maven.remote-repositories.repo1.url=http://repo.spring.io/libs-snapshot`  For example:

```
stream create foo --definition "triggertask --uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.1.RELEASE --fixed-delay=5 | task-launcher-local --maven.remote-repositories.repo1.url=http://repo.spring.io/libs-snapshot" --deploy
```